src/components/policy/policy_regular/include/policy/policy_table/types.h

```diff
@@ -67,7 +67,7 @@ typedef Array<Enum<HmiLevel>, 0, 4> HmiLevels;
 
 typedef Array<Enum<Parameter>, 0, 24> Parameters;
 
-typedef Map<RpcParameters, 0, 50> Rpc;
+typedef Map<RpcParameters, 0, 65535> Rpc;
```

Comment:
@AKalinich-Luxoft it really should be so big? If yes, could you please use some platform specific constant of intmax?